tests/goats_tom/serializers/gpp/test_scheduling_windows.py

```diff
@@ -0,0 +1,224 @@
+import pytest
+from rest_framework.exceptions import ValidationError
+
+from goats_tom.serializers.gpp.schedulingWindows import (
+    TimingWindowAfterSerializer,
+    TimingWindowRepeatPeriodSerializer,
+    TimingWindowRepeatSerializer,
+    TimingWindowEndSerializer,
+    TimingWindowSerializer,
+    SchedulingWindowsSerializer,
+)
+from gpp_client.api.enums import TimingWindowInclusion
+
+
+#
+# TimingWindowAfterSerializer
```

Comment:
General comment for the tests, you could look into parametrizing them, as many of these tests are small and just testing different input values. It would allow you to reduce how much code you repeat and make it easier to test even more possible options.